scripts/CSSRE/rolling-restart-brokers/metadata.yaml

```diff
@@ -0,0 +1,43 @@
+file: rolling-restart-brokers.sh
+name: rolling-restart-brokers
+description: Manually roll the Kafka brokers
+author: Pat Cremin
+allowedGroups:
+  - CSSRE
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "apps"
+      resources:
+        - "statefulsets"
```

Comment:
Why it's clusterRoleRules here? can it be namespace scoped?

Reply:
I'd assumed from testing it needed clusterRoles, as other CSSRE examples were using cluster-wide roles.  I can see the example in SREP and will test and then switch it.

Reply:
Actually, there is no fuzzy logic on the namespace limitation, so it would need to remain as  clusterRoleRules, as the namespace is unknown and requested as part of the script. unless there is a way to feed the provided namespace into the metadata after the fact, I can't see how I could specify the NS.